src/isotools/gene.py

```diff
@@ -677,33 +678,33 @@ def filter_transcripts(self, query=None, min_coverage=None, max_coverage=None):
             assert all(f in tr_filter for f in used_tags), msg.format(
                 ', '.join(f for f in used_tags if f not in tr_filter), ', '.join(tr_filter))
             tr_filter_fun = {tag: _filter_function(tag, tr_filter)[0] for tag in used_tags if tag in tr_filter}
```

Comment:
I guess it should be change here if I understand correctly:
`tr_filter_func = {tag: _filter_function(transcript_filter[tag])[0] for tag in used_tags if tag in transcript_filter}`